CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Because of the version bump to `go`, the macOS build for this release requires a
 1. [21977](https://github.com/influxdata/influxdb/pull/21977): Flux metaqueries for `_field` take fast path if `_measurement` is the only predicate.
 1. [22060](https://github.com/influxdata/influxdb/pull/22060): Copy names from mmapped memory before closing iterator
 1. [22052](https://github.com/influxdata/influxdb/pull/22052): systemd service -- handle 40x and block indefinitely
+1. [22236](https://github.com/influxdata/influxdb/pull/22236): influxdb2 packages should depend on curl
```

Comment:
Agh sorry, this also needs to get bumped up under the `2.0.9` section I think

Reply:
ah true